columnq/src/table/mod.rs

```diff
@@ -140,6 +141,7 @@ pub enum TableLoadOption {
     parquet {},
     google_spreadsheet(TableOptionGoogleSpreasheet),
     delta {},
+    arrow_stream {},
```

Comment:
how about we keep the name as `arrows` to be consistent with other file formats?